src/containers/VisualElement/VisualElementSearch.jsx

```diff
@@ -150,7 +150,7 @@ class VisualElementSearch extends Component {
       case 'h5p': {
         return (
           <Fragment>
-            <h2>{titles(t, selectedResource)[selectedResource]}</h2>
+            <h1>{titles(t, selectedResource)[selectedResource]}</h1>
```

Comment:
H2 har styling med `margin-top: ${spacing.large}` i ModalBody og da gjør det ikke så stor forskjell å fjerne headeren (ref bilde), så byttet til H1. Men stylingen på H1 gir blå farge (`colors.brand.primary`), så hvis den skal beholdes som svart må jeg fikse det i frontend-packages.

<img width="1716" alt="Skjermbilde 2021-04-13 kl  10 13 58" src="https://user-images.githubusercontent.com/55005434/114532360-10035200-9c4d-11eb-9416-8a3b3d42ec1e.png">

Reply:
Trur ikjke det er veldig farlig om header er blå. 
Dersom vi skal ha mindre rammer på modalen kan vi ikkje bruke ndla/modal og det har eg ikkje så lyst til. Men kan du sjå om du får endra høgda til å ta utgangspunkt i viewModal? Litt som det som er gjort på H5PPage?

Reply:
Sånn at h5p-komponenten dekker mer av modalen mener du?

Reply:
Kan sette min-height til typ 85vh elns, men problemet er at `VisualElementSearch` også blir brukt i `VisualElementSelectField` i topicarticle editor, og der ligger den i en lightbox som ikke er like stor som modalen. Men kan eventuelt endre den til å bruke ndla/modal så den blir seende lik ut som når den brukes i slatepicker?

Reply:
Ja, kunne brukt modal i stedet for lightbox fra slate-picker. Det er en mulighet.